Coding/server/src/WiFiManager/WiFiManager.h

```diff
@@ -101,36 +97,36 @@ class WiFiManager
 
 private:
     /** Reference to store */
-    Store &m_store = Store::getInstance();
+    Store &m_store;
```

Comment:
Please use a unified order within the header files

Reply:
Fixed